model/src/main/java/org/cbioportal/genome_nexus/model/AnnotationType.java

```diff
@@ -0,0 +1,36 @@
+package org.cbioportal.genome_nexus.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+
+public enum AnnotationType {
```

Comment:
Not sure if `Type` is the right name for this enum. `Field` may not be the best name either but maybe for now keep it as is?

```suggestion
public enum AnnotationField {
```